tf2_geometry_msgs/CMakeLists.txt

```diff
@@ -46,5 +46,5 @@ install(DIRECTORY include/${PROJECT_NAME}/
 #
 #
 # endif()
-ament_export_include_directories(include)
-ament_package()
+ament_auto_find_build_dependencies()
```

Comment:
thank you for the review! I had it here since I was just using it for getting [these variables defined](https://github.com/ament/ament_cmake/blob/c65dd2dfae39c8772580a3f40a46044fda34c98f/ament_cmake_auto/cmake/ament_auto_package.cmake#L34) but it makes sense to use it as intended at the top: 
031d69ed4569228127a27c49669d3ee18ce4624f

https://github.com/ros2/geometry2/pull/9/commits/b0bce4f4fe4ffc28cf0c4ff0552f75bec6f607f4 checks that the packages were found with `<pkgname>_FOUND` as suggested.